src/vm/core.zig

```diff
@@ -42,16 +42,15 @@ pub const CairoVM = struct {
     /// - `CairoVM`: The created VM.
     /// # Errors
     /// - If a memory allocation fails.
-    pub fn init(allocator: *const Allocator, config: Config) !CairoVM {
+    pub fn init(comptime capacity: usize, allocator: *const Allocator, config: Config) !CairoVM {
```

Comment:
please change to a more explicit name, it's more the `trace_entries_initial_capacity`
And I would put it after config in the orders of parameters.

Reply:
@abdelhamidbakhta Sounds good, just pushed the change.